db/test/test_stats.py

```diff
@@ -16,16 +14,16 @@
 class StatsTestCase(unittest.TestCase):
     """Statistics methods which use mocked database methods for testing"""
 
-    def test_get_next_hour(self):
-        date1 = datetime.datetime(2016, 01, 07, 10, 20, 39, tzinfo=pytz.utc)
-        next_hour = db.stats._get_next_hour(date1)
-        expected = datetime.datetime(2016, 01, 07, 11, 0, 0, tzinfo=pytz.utc)
-        self.assertEqual(next_hour, expected)
+    def test_get_next_day(self):
+        date1 = datetime.datetime(2016, 1, 7, 10, 20, 39, tzinfo=pytz.utc)
+        next_day = db.stats._get_next_day(date1)
+        expected = datetime.datetime(2016, 1, 8, 0, 0, 0, tzinfo=pytz.utc)
+        self.assertEqual(next_day, expected)
 
-        date2 = datetime.datetime(2016, 01, 07, 13, 0, 0, tzinfo=pytz.utc)
-        next_hour = db.stats._get_next_hour(date2)
-        expected = datetime.datetime(2016, 01, 07, 14, 0, 0, tzinfo=pytz.utc)
-        self.assertEqual(next_hour, expected)
+        date2 = datetime.datetime(2016, 1, 7, 13, 0, 0, tzinfo=pytz.utc)
```

Comment:
todo: this test was specifically checking that h:00:00 would result in h+1 for the hour. It doesn't make much sense to have these two tests when skipping days. Perhaps a check at midnight for this test